packages/lesswrong/components/posts/TableOfContents/TableOfContentsDrawerRow.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { PureComponent, Component } from 'react';
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from 'material-ui/MenuItem';
```

Comment:
Use the new Material UI MenuItem (this also allows you to get rid of the !important below)